steps/appdome-build-2secure-android/1.0.1/step.yml

```diff
@@ -0,0 +1,52 @@
+title: Appdome Build
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API.
+website: https://github.com/Appdome/bitrise_build-2secure-android
+source_code_url: https://github.com/Appdome/bitrise_build-2secure-android
+support_url: https://github.com/Appdome/bitrise_build-2secure-ios/issues
```

Comment:
The source code url is Android but the support url is iOS specific.

Reply:
fix done.